README.md

```diff
@@ -1,68 +1,158 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+[![Build Status](https://api.travis-ci.com/zmgalante/lletter6.svg?branch=master)](https://travis-ci.com/github/zmgalante/lletter6)
+[![Cypress.io](https://img.shields.io/badge/tested%20with-Cypress-04C38E.svg)](https://www.cypress.io/)
 
-## Available Scripts
+# Summary
+
+This project is intended for personal use only, project made for fun between friends. This is a simple implementation of the board game `Love Letter` the 6-player version. We do not own this game this game is made by **[Alderac Entertainment Group](https://www.alderac.com/)**.
+
+# Table of Contents
+
+- [Contribution Workflow](#contribution-workflow)
+	- [Issues](#issues) 
+	- [Branches](#branches)
+	- [Pull Requests](#pull-requests)
+- [Setup Project](#setup-project)
+- [Developing](#developing)
+	- [Scripts](#scripts)
+	- [Unit Tests](#unit-tests)
+	- [E2E Tests](#e2e-tests)
+
+<br />
+
+## Contribution Workflow
+
+Make sure you follow the steps bellow before picking up and issue and start developing.
+
+### Issues
+
+If there is already an issue created for something you want to resolve/develop, make sure to assing that issue to yourself, otherwise please create an issue before start developing.
+<br />
+<br />
+If you have to create an issue there are two types:
+
+| Type          | When                               | Labels            |
+| :------------ | :--------------------------------- | :---------------- |
+| Feature       | there is no support in the project | enhancement       |
+| Bugfix        | there is faulty implementation     | help wanted, bug  |
+
+***Note: Please make sure you fill these issue templates the most detailed possible***
+
+### Branches
+
+When creating a new branch please make sure this one is created from master and ***contains the right prefix***, like so:
+
+```
+issue: Feature - Add reach router #16
+branch-name: feature/add-reach-router
+---
+issue: Bugfix - Reach router not properly working #18
+branch-name: bugfix/fix-reach-router
+```
+***Note: When you are finished and the changes are in master please close the branch***
+
+### Pull Requests
+
+When commiting your changes you will notice that when you do `git commit` in your branch a script will run, this script will ensure that you respect our `eslint` and `prettier` rules before letting you commit and push anything.
```

Comment:
This reminds me, is there a convention for commits? Like if I'm working on issue 15 should I put something like "[RP-15] Added new amazing feature that will make this love letter better than others being made right now"?

Reply:
Don't think we need to do that here, when you merge the PR if the issue is linked the commit message will contain (#number) at the end and should be enough :) just make sure you link the issue when opening a PR it is the first item on the checklist and contains a link to the info you need :)